.scripts/util.sh

```diff
@@ -252,8 +265,9 @@ copy_ehci_usb_modules() {
 
 copy_ct60_usb_modules() {
 	local USBDIR="$1"
+	local TARGET="$2"
```

Comment:
This is not needed. CT60 (and therefore Ethernat) always implies 68060. It just adds confusion why *CT60* function has to be called with "060".

Reply:
Huh. I haven't realised the effect of `deb) TEMP_CPU_TARGET=02060` until now.

In that case this is still inconsistent with other machine-specific modules, isn't it? E.g. for the TT we always use `030`, so why suddenly `${TARGET}` here?

I think `deb` target shouldn't change / override CPU definitions, it's quite confusing. Maybe it could be transformed into a flag, i.e. if building with `DEBUG=1`, add proper `CFLAGS` etc.

Reply:
That won't solve the problem. You need to use a different .compile_dir when compiling the same source with different settings (.compile_deb in this case).

Using the target to override different settings is also used in other places, mainly the sockets/xif directory where the lance driver is compiled for st/mste etc.

Reply:
Ok but that still doesn't answer my question why did you choose this specific function to and not others, where the CPU is hardcoded as well?

Reply:
Because the deb target is not used in all subdirectories so far.

The deb target is a bit of a hack anyway. Until now it was not even build by the scripts (see also the commit for the scc driver where i had to fix some printfs). It is also not deployed. And in some directories, the CPU type for it is changed to 030. I kept it this way because i wasn't sure about the reason for it.

Reply:
OK, I create a new issue where this can be discussed.